heroic-test/src/main/java/com/spotify/heroic/test/AbstractMetricBackendIT.java

```diff
@@ -82,7 +83,7 @@
      * See: https://github.com/spotify/heroic/pull/208 */
     protected boolean brokenSegmentsPr208 = false;
     protected boolean eventSupport = false;
-    protected Optional<Integer> maxBatchSize = Optional.empty();
```

Comment:
Why the replacement?

Reply:
Just doing some cleanup as I went through the code. While I generally like Optional, in this case its only purpose is a single assume statement in the test, so removing it makes things a tiny bit easier to read/less verbose.